gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -228,7 +228,7 @@ public void setDevices(List<DeviceCaptureInfo> devices) {
 
     @Override
     public String getTitle() {
-      return Messages.CAPTURE_TRACE_GRAPHICS;
+      return Messages.CAPTURE_TRACE_PERFETTO;
```

Comment:
This is the wrong "fix". When the "API" dropdown changes, we update the title:
```
getShell().setText(
            isPerfetto ? Messages.CAPTURE_TRACE_PERFETTO : Messages.CAPTURE_TRACE_GRAPHICS);
```
We need to make sure that this is also done when first initializing it. Otherwise we will show the wrong title again if for a user graphics trace is selected at init time. Maybe here we should give a default title of "Capture Trace" and then change it on dropdown init and selection change.

Reply:
I chose to display a default "select trace type" message on dialog creation, then the code you pointed out will update the dialog title when a given trace type is selected.